pkgs/development/interpreters/supercollider/default.nix

```diff
@@ -33,11 +33,17 @@ stdenv.mkDerivation rec {
 
   dontWrapQtApps = true;
```

Comment:
Would it also work to simply remove this, and don't have a `postFixup` phase?

Reply:
I just tested locally, yes it does.

Reply:
That'll work too, whoops. Only difference is that it'll wrap some binaries that don't use Qt in any capacity, not that it really matter I guess.

Reply:
True, I don't know whether that's an issue